### ISSUE TITLE
Is there a pre-trained model here?

### ISSUE THREAD

**Issue by JJ-Guo1996** [s3prl/s3prl#351]

**Comment by andi611** [comment]
Yes, [upstream models](https://github.com/s3prl/s3prl/tree/master/s3prl/upstream) are all pre-trained and stored in the cloud, run the provided commands of downstream tasks and they will be downloaded for you.

Or you can look at the `hubconf.py` file under each upstream model's directory to find the download link. For example:
https://github.com/s3prl/s3prl/blob/540a4f86e9f099b240c314a56ff80b069e91d4cd/s3prl/upstream/tera/hubconf.py#L153-L164
